diff --git a/drivers/platform/surface/surface_aggregator_registry.c b/drivers/platform/surface/surface_aggregator_registry.c
index aeb3feae4..dd34a6bc5 100644
--- a/drivers/platform/surface/surface_aggregator_registry.c
+++ b/drivers/platform/surface/surface_aggregator_registry.c
@@ -290,7 +290,6 @@ static const struct software_node *ssam_node_group_sp8[] = {
 	&ssam_node_tmp_pprof,
 	&ssam_node_kip_tablet_switch,
 	&ssam_node_hid_kip_keyboard,
-	&ssam_node_hid_kip_penstash,
 	&ssam_node_hid_kip_touchpad,
 	&ssam_node_hid_kip_fwupd,
 	&ssam_node_hid_sam_sensors,
